src/Annotation/AsymmetricKeysGenerator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\pubkey_encrypt\Annotation\AsymmetricKeysGenerator.
+ */
+
+namespace Drupal\pubkey_encrypt\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Asymmetric Keys Generator annotation object.
+ *
+ * @see \Drupal\pubkey_encrypt\Plugin\AsymmetricKeysManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class AsymmetricKeysGenerator extends Plugin {
```

Comment:
You do need to add the "@ingroup" annotation, but class names do not require module name prefixing since it's namespaced to the module